### PR TITLE
Bid Type Count Explanations

### PR DESCRIPTION
**Pull request by mjoyce91**
Addresses #755 by displaying the bid counts on position results cards and details pages. Integrates with [react-tippy](https://tvkhoa.github.io/testlib/), a tooltip module, to show bid definitions within 
a tooltip on hover.

### CODE REVIEW

File: src/sass/_tooltip.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.tippy-tooltip--regular {
```
Comment (burgwyn) [inline src/sass/_tooltip.scss]: Add a comment that this is a vendor override

File: src/sass/_results.scss [hunk 2]
```diff
@@ -78,6 +78,14 @@
   margin-bottom: 1em;
   padding: 1em 1.2em;
 
+  .bid-count-container {
+    float: right;
+  }
+
+  .tippy-tooltip {
```
Comment (burgwyn) [inline src/sass/_results.scss]: Is this vendor override?

Comment (mjoyce91) [reply src/sass/_results.scss]: This was unneeded

File: src/Components/PositionTitle/PositionTitle.jsx [hunk 3]
```diff
@@ -108,6 +108,8 @@ class PositionTitle extends Component {
 
     const isAllowedToEdit = !!(details.description && details.description.is_editable_by_user);
 
+    const shouldShowBidStats = !!details.bid_statistics && !!details.bid_statistics[0];
```
Comment (burgwyn) [inline src/Components/PositionTitle/PositionTitle.jsx]: 👍

**Pull request merged**